src/dsp_tools/dsp_tools.py

```diff
@@ -166,77 +166,82 @@ def program(user_args: list[str]) -> None:
-                               user_mail=args.user,
-                               password=args.password,
-                               verbose=args.verbose,
-                               dump=args.dump if args.dump else False)
+                success = create_project(project_file_as_path_or_parsed=args.projectfile,
+                                         server=args.server,
+                                         user_mail=args.user,
+                                         password=args.password,
+                                         verbose=args.verbose,
+                                         dump=args.dump if args.dump else False)
     elif args.action == "get":
-        get_project(project_identifier=args.project,
-                    outfile_path=args.projectfile,
-                    server=args.server,
-                    user=args.user,
-                    password=args.password,
-                    verbose=args.verbose)
+        success = get_project(project_identifier=args.project,
+                              outfile_path=args.projectfile,
+                              server=args.server,
+                              user=args.user,
+                              password=args.password,
+                              verbose=args.verbose)
     elif args.action == "xmlupload":
         if args.validate:
-            validate_xml_against_schema(input_file=args.xmlfile)
+            success = validate_xml_against_schema(input_file=args.xmlfile)
         else:
-            xml_upload(input_file=args.xmlfile,
-                       server=args.server,
-                       user=args.user,
-                       password=args.password,
-                       imgdir=args.imgdir,
-                       sipi=args.sipi,
-                       verbose=args.verbose,
-                       incremental=args.incremental,
-                       save_metrics=args.metrics)
+            success = xml_upload(input_file=args.xmlfile,
+                                 server=args.server,
+                                 user=args.user,
+                                 password=args.password,
+                                 imgdir=args.imgdir,
+                                 sipi=args.sipi,
+                                 verbose=args.verbose,
+                                 incremental=args.incremental,
+                                 save_metrics=args.metrics)
     elif args.action == "excel2json":
-        excel2json(data_model_files=args.data_model_files,
-                   path_to_output_file=args.outfile)
+        success = excel2json(data_model_files=args.data_model_files,
+                             path_to_output_file=args.outfile)
     elif args.action == "excel2lists":
-        excel2lists(excelfolder=args.excelfolder,
-                    path_to_output_file=args.outfile,
-                    verbose=args.verbose)
+        _, success = excel2lists(excelfolder=args.excelfolder,
```

Comment:
is there a reason for the first return value, if you discard it anyways?

Reply:
Yes, the method is used elsewhere, where the first return value is used